lib/schemes/flat.js

```diff
@@ -6,5 +6,15 @@ module.exports = {
         var naming = bemNaming(options.naming);
 
         return naming.stringify(entity) + '.' + tech;
+    },
+    parse: function(path, options) {
+        options || (options = {});
+        var naming = bemNaming(options.naming),
+            splittedPath = path.split('.');
+
+        return {
```

Comment:
I don't understand profit of such naming so such simple things. Can't we refactor it later when we end up with some name and found real cases for it?